### PR TITLE
[Merged by Bors] - feat: Unions and intersections indexed by `Finset.sigma`

### PR DESCRIPTION
**Pull request by YaelDillies**
Followup to #8964

From PFR

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Fintype/Sigma.lean [hunk 1]
```diff
@@ -36,3 +36,15 @@ instance PSigma.fintype {α : Type*} {β : α → Type*} [Fintype α] [∀ a, Fi
     Fintype (Σ'a, β a) :=
   Fintype.ofEquiv _ (Equiv.psigmaEquivSigma _).symm
 #align psigma.fintype PSigma.fintype
+
+lemma Set.biUnion_finsetSigma_univ (s : Finset ι) (f : Sigma κ → Set α) :
+    ⋃ ij ∈ s.sigma fun _ ↦ Finset.univ, f ij = ⋃ i ∈ s, ⋃ j, f ⟨i, j⟩ := by aesop
+
+lemma Set.biUnion_finsetSigma_univ' (s : Finset ι) (f : ∀ i, κ i → Set α) :
```
Comment (riccardobrasca) [inline Mathlib/Data/Fintype/Sigma.lean:43]: I am not a big fan of the notatiion `f : ∀ i, κ i → Set α`, but I guess `PiNotation` is not available here.

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/Data/Fintype/Sigma.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Data/Fintype/Sigma.lean]: It's available, actually, but I'm afraid using it is a lost cause. I can only find it in comments.

**Comment by YaelDillies** [comment]
bors mereg

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**